app/scripts/CenterTrack.js

```diff
@@ -459,15 +460,16 @@ class CenterTrack extends React.Component {
         }}
         styleName="styles.center-track"
       >
-        {isBrushable &&
-          <svg
+        {isBrushable
+          && (
+<svg
```

Comment:
Is this really the level of indentation the linter expects?

Reply:
I think it's what `--fix` gave me... but it still lints with the indentation we'd expect, so I'll fix this.